R/variable-to-R.R

```diff
@@ -148,6 +148,12 @@ paginatedGET <- function (url, query, offset=0, limit=1000, table=FALSE) {
 #' something beyond what is currently supported, you can bring a variable's
 #' data into R with `as.vector(ds$var)` and work with it like any
 #' other R vector.
+#' 
+#' `as.vector` transfers data from Crunch to a local R session. Note: 
+#' `as.vector` returns the vector in the row order of the dataset. If filters 
+#' are set that specify an order that is different from the row order of the 
+#' dataset, the results will ignore that order. If you need the vector ordered 
+#' in that way, use syntax like `as.vector(ds$var)[c(10, 5, 2)]` instead.
```

Comment:
It wouldn't be impossible to make `as.vector` do the right thing. `</scope creep>` (Feel free to make an issue for it.)